### PR TITLE
LP-2277 Fix all linting issues of Classrooms application

### PR DESCRIPTION
**Pull request by aliassad**
### Description

[LP-2277](https://philanthropyu.atlassian.net/browse/LP-2277)

### Notes
I have fixed the linting issues related to classrooms application, In which i have added missing doc strings and removed couple of unused imports. 

### Final Report
`
-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 6.30/10, +1.11)

`

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/classrooms/models.py [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 from django.db import models
-from nodebb.models import DiscussionCommunity
 from model_utils.models import TimeStampedModel
 
+from nodebb.models import DiscussionCommunity
+
 
 class DiscussionCommunityMembership(TimeStampedModel):
+    """
+    A class to associated user with community
```
Comment (zahid-arbisoft) [inline openedx/features/classrooms/models.py:13]: `associate`

File: openedx/features/classrooms/apps.py [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+"""
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
 from django.apps import AppConfig
 
 
 class ClassroomsConfig(AppConfig):
+    """
+    A Default Config class for classrooms application
```
Comment (zahid-arbisoft) [inline openedx/features/classrooms/apps.py:12]: `A default config`

Also can we use any other work rather than config

**Review by zahid-arbisoft — CHANGES_REQUESTED** [review]

File: openedx/features/classrooms/apps.py (hunk 2 above)
Comment (aliassad) [reply openedx/features/classrooms/apps.py]: Added

File: openedx/features/classrooms/models.py (hunk 1 above)
Comment (aliassad) [reply openedx/features/classrooms/models.py]: chaged

**Review by zahid-arbisoft — APPROVED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/classrooms/admin.py [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+"""
 from django.contrib import admin
 
-from models import DiscussionCommunityMembership
+from .models import DiscussionCommunityMembership
 
 
 class DiscussionCommunityMembershipAdmin(admin.ModelAdmin):
+    """
+    A Admin class for discussion community membership
```
Comment (harris-irfan) [inline openedx/features/classrooms/admin.py:11]: Just "Admin class for discussion community membership" would be fine. Please remove "A "

File: openedx/features/classrooms/apps.py [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+"""
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
 from django.apps import AppConfig
 
 
 class ClassroomsConfig(AppConfig):
+    """
+    A default configurations for classrooms application
```
Comment (harris-irfan) [inline openedx/features/classrooms/apps.py:12]: Let's change it to: "Default configuration for classrooms application"

File: openedx/features/classrooms/models.py [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
+    A class to associate user with community
+    """
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     community = models.ForeignKey(DiscussionCommunity, on_delete=models.CASCADE)
 
     class Meta:
         unique_together = (("user", "community"),)
+
+    def __unicode__(self):
+        return u"<DiscussionCommunityMembership: {user} >".format(user=self.user)
```
Comment (harris-irfan) [inline openedx/features/classrooms/models.py:22]: single qoutes?

**Review by harris-irfan — CHANGES_REQUESTED** [review]

File: openedx/features/classrooms/admin.py (hunk 3 above)
Comment (aliassad) [reply openedx/features/classrooms/admin.py:11]: Done

File: openedx/features/classrooms/apps.py (hunk 4 above)
Comment (aliassad) [reply openedx/features/classrooms/apps.py]: Done

File: openedx/features/classrooms/models.py (hunk 5 above)
Comment (aliassad) [reply openedx/features/classrooms/models.py]: Changed

**Review by harris-irfan — APPROVED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**